scripts/install-iqsharp.ps1

```diff
@@ -5,12 +5,27 @@ $ErrorActionPreference = 'Stop'
 & "$PSScriptRoot/set-env.ps1"
 
 # Install iqsharp if not installed yet.
-dotnet iqsharp --version
-If ($LastExitCode -ne 0) {
+
+Write-Host ("Installing IQ# tool.")
+$install = $False
+
+# Install iqsharp if not installed yet.
+try {
+    dotnet iqsharp --version
+    $install = ($LastExitCode -ne 0)
+} catch {
+    Write-Host ("`dotnet iqsharp --version` threw Exception.")
```

Comment:
Do we want to write out what the exception was in this case?

Reply:
I added logging the error message in this case similar to the other scenario.